.github/workflows/ci.yml

```
@@ -0,0 +1,99 @@
+      - name: Check Go version
+        run: go version
+
+      - name: Install Gosec
+        run: go install github.com/securego/gosec/v2/cmd/gosec@latest
+
+      - name: Run Gosec (tolerate findings)
+        continue-on-error: true
+        run: |
+          set -euo pipefail
+          gosec -fmt sarif -out gosec-results.sarif ./...
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)